pkg/utils/configMapUtils.go

```diff
@@ -13,51 +13,46 @@ func (expDetails *ExperimentDetails) PatchConfigMaps(clients ClientSets, engineD
 		return err
 	}
 
-	log.Info("Validating configmaps specified in the ChaosExperiment & ChaosEngine")
-
-	if err := expDetails.ValidateConfigMaps(clients); err != nil {
-		return err
+	if len(expDetails.ConfigMaps) != 0 {
+		log.Info("Validating configmaps specified in the ChaosExperiment & ChaosEngine")
+		if err := expDetails.ValidateConfigMaps(clients); err != nil {
+			return err
+		}
 	}
-
 	return nil
 }
 
 // ValidateConfigMap validates the configMap, before checking or creating them.
-func (clientSets ClientSets) ValidateConfigMap(configMapName string, experiment *ExperimentDetails) error {
-
-	_, err := clientSets.KubeClient.CoreV1().ConfigMaps(experiment.Namespace).Get(configMapName, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	return nil
-
+func (clientSets ClientSets) ValidateConfigMap(configMapName, namespace string) error {
```

Comment:
To make it more explicit and meaningful and to differentiate from schema validation can we call it `ValidatePresenceOfConfigMapResourceInCluster`